package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "react-native-keyboard-manager",
-	"version": "4.0.13-16",
+	"version": "6.5.4-17",
 	"description": "Library that allows to prevent issues of keyboard sliding up and cover on React-Native iOS projects. ⚛",
 	"private": false,
```

Hey! Don't forget about enabling typescript types here ;)

```suggestion
	"private": false,
	"types": "index.d.ts",
```